src/service_client/assisted_service_api.py

```diff
@@ -106,8 +106,13 @@ def wait_for_api_readiness(self, timeout: int) -> None:
     def create_cluster(
         self, name: str, ssh_public_key: Optional[str] = None, **cluster_params
     ) -> models.cluster.Cluster:
-        cluster = models.ClusterCreateParams(name=name, ssh_public_key=ssh_public_key, **cluster_params)
-        log.info("Creating cluster with params %s", cluster.__dict__)
+        cluster = models.ClusterCreateParams(name=name, ssh_public_key=ssh_public_key, **cluster_params).to_dict()
+
+        # TODO remove after completely removing `user_managed_networking` from cluster object
```

Comment:
how will the service act after the change? will it ignore this parameter or it won't even access this unrecognized parameter?
the first option might be a slightly better option because we at least allow a gradual transition iiuc (clients are transitioning from this parameter to just setting the platform)

Reply:
We actually don't need this parameter at all. 
The service currently set it automatically by a given platform if provided, and if not baremetal platform is the default one.
The first step (after removing the dependencies - e.g. test-infra, UI) is to remove the logic but to keep the parameter passing UMN to any API call won't fail but will do nothing

Reply:
So what's the value in removing it here? iiuc it doesn't do a whole lot rather than removing this parameter under the hood (and the only place that really matters is the outer scope of this function, e.g. QE tests, test-infra tests, etc.)